composites/basic/SubNavigation.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import styled from "styled-components";
+import colors from "../../style-guide/colors.json";
+import PropTypes from "prop-types";
+
+/**
+ * Defines the styling of the SubNavigationElement.
+ */
+const SubNavigationElement = styled.nav`
+	border-bottom: 1px solid ${ colors.$color_grey_light };
+	font-size: 1em;
+	text-transform: uppercase;
+	display: block;
+	
+	ul {
```

Comment:
ul Should be a separate styled component, instead of nested inside SubNavigatonElement